src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx

```diff
@@ -82,38 +83,47 @@ class RoutedSavableReviewPage extends React.Component {
       return saveLink;
     }
 
-    let InlineErrorComponent;
-    if (typeof errorText === 'function') {
-      InlineErrorComponent = errorText;
-    } else if (typeof errorText === 'string') {
-      InlineErrorComponent = () => <p>{errorText}</p>;
-    } else {
-      InlineErrorComponent = () => (
-        <p>
-          If it still doesn’t work, please <CallHRC />
-        </p>
-      );
-    }
-
-    return (
-      <div className="usa-alert usa-alert-error schemaform-failure-alert">
-        <div className="usa-alert-body">
-          <p className="schemaform-warning-header">
-            <strong>We’re sorry. We can't submit your form right now.</strong>
-          </p>
+    const DefaultErrorMessage = () => {
+      let InlineErrorComponent;
+      if (typeof errorText === 'function') {
+        InlineErrorComponent = errorText;
+      } else if (typeof errorText === 'string') {
+        InlineErrorComponent = () => <p>{errorText}</p>;
+      } else {
+        InlineErrorComponent = () => (
           <p>
-            We’re working to fix the problem. Please make sure you’re connected
-            to the Internet, and then try saving your form again. {saveLink}.
+            If it still doesn’t work, please <CallHRC />
           </p>
-          {!user.login.currentlyLoggedIn && (
+        );
+      }
+
+      return (
+        <div className="usa-alert usa-alert-error schemaform-failure-alert">
+          <div className="usa-alert-body">
+            <p className="schemaform-warning-header">
+              <strong>We’re sorry. We can't submit your form right now.</strong>
+            </p>
             <p>
-              If you don’t have an account, you’ll have to start over. Try
-              submitting your form again tomorrow.
+              We’re working to fix the problem. Please make sure you’re
+              connected to the Internet, and then try saving your form again.{' '}
+              {saveLink}.
             </p>
-          )}
-          <InlineErrorComponent />
+            {!user.login.currentlyLoggedIn && (
```

Comment:
Are we concerned about Law of Demeter issues here? Will user.login always be present with data type of object?

Reply:
I honestly am not sure to be honest with you, this was code that was already there that I just moved into a function. Perhaps when a VSP member can answer this question when they review this PR. If it is not always there more than willing to throw some `optional chaining` in there.

Reply:
I'm ok leaving it as it. There are tons of places in the code where we call `user.login.currentlyLoggedIn` with impunity, so it must be safe to do so.